Base/QTCore/qSlicerAbstractCoreModule.cxx

```diff
@@ -342,6 +342,8 @@ QString qSlicerAbstractCoreModule::defaultDocumentationLink()const
     return "";
     }
   QString url = app->moduleDocumentationUrl(this->name());
-  QString linkText = QString("<p>For more information see the <a href=\"%1\">online documentation</a>.</p>").arg(url);
-  return linkText;
+  QString linkText = QString("<p>")
+    + tr("For more information see the %1.").arg(
+        QString("<a href=\"%1\">%2</a>").arg(url).arg(tr("online documentation")))
+    + QString("</p>");
```

Comment:
For reference, the `return` statement was missing and this was causing all tests starting Slicer to fail. Additionally, the preview build was also failing to start.

This is now fixed in https://github.com/Slicer/Slicer/pull/6793